### ISSUE TITLE
[Bug] What is happening?

### ISSUE THREAD

**Issue by mobsenpai** [AppFlowy-IO/AppFlowy#7070]
### Bug Description

The tables are not showing up only the kanban views work.

### How to Reproduce

Only happens with Android app

### Expected Behavior

The tables/grids show properly

### Operating System

Android

### AppFlowy Version(s)

0.7.8

### Screenshots

![Screenshot_2024-12-27-22-46-08-21_42ce4c28eaf7aa554cc10b03f29ef10b](https://github.com/user-attachments/assets/280850a5-c5fd-4409-a9d8-7f31ee519d67)

**Comment by appflowy** [comment]
Hi @mobsenpai Have you tried to reopen the page?

**Comment by mobsenpai** [comment]
Yes I have. I also tried reinstalling it only happens on the reference grids view I think as I can open the individual grids files.

**Comment by appflowy** [comment]
@mobsenpai Could you please remove the filter and check again? Some rows might have been filtered out.

**Comment by mobsenpai** [comment]
@appflowy yes I checked it. I am not able to remove the filters, so I made a fresh grid inside a document and it's the same result.
